crates/project/src/project.rs

```diff
@@ -3201,6 +3201,51 @@ impl Project {
         let disk_based_diagnostics_progress_token =
             adapter.disk_based_diagnostics_progress_token.clone();
 
+        // Log LSP messages under experimental/serverStatus and display pop-up for errors
+        language_server
+            .on_notification::<ServerStatus, _>({
+                let this = this.clone();
+                let name = name.to_string();
+                move |params, mut cx| {
+                    let this = this.clone();
+                    let name = name.to_string();
+                    match params.health {
+                        ServerHealthStatus::Ok => log::info!("Health: Ok"),
```

Comment:
> log::info!("Health: Ok")

Sorry, this is the first time I properly concentrated on the code, and now thinking my advice about logging this particular part does not make much sense.
I would propose to check for message presence and only log things if there's anything there.
Otherwise, adding a random "Health: Ok" into the logs does not seem very informative.